mozilla-iam/gsuite-community-drive-driver
# add drive archiving methods

* Adds methods to archive drives.

Testing plan:

* Create a mozillians group
* Add members
* Ensure they are added to the team drive.
* Remove all members and delete the group
* All members should be removed from the drive. 
* Drive will be renamed to archived_%TIMESTAMP%_%DRIVENAME%_mozilliansorg 
* The connector should block the creation of a new drive for a group with the exact same name until an admin sets a flag in DynamoDb to allow the name to be reused.
